docs/reference/faq/index.md

```diff
@@ -131,4 +211,651 @@ for more details.
+
+Yes, the Kafka consumer group name can be changed by specifying the following configuration **SOURCE_KAFKA_CONSUMER_GROUP_ID**. The Kafka consumer in Firehose can also be reset. However, the default behaviour is
+currently set to read from the latest offset. To know further details on how to tune the Kafka consumer group, you can
+have a look at the Kafka consumer configurations already exposed by
+Firehose [here](https://github.com/odpf/firehose/blob/main/docs/reference/configuration/kafka-consumer-1.md#kafka-consumer).
+
+#### What are the advantages of Firehose over Kafka connect ?
+
+- **Ease of use:**  Firehose is easier to install, and using different sinks only requires changing a few
+  configurations. However, Kafka Connect requires connectors to be installed across all the worker nodes within the
+  cluster when used in distributed mode.
+- **Filtering:**  Value-based filtering is much easier to implement as compared to Kafka Connect. Requires no additional
+  plugins/schema-registry to be installed.
+- **Extensible:**  Provides a comprehensible abstract sink contract making it easier to add a new sink in Firehose.
+  Firehose also comes with an inbuilt serialisation/deserialisation and doesn't require any converters and serialisers
+  when implementing a new sink.
+- **Easy monitoring:**  Firehose provides a detailed health dashboard (Grafana) for effortless monitoring.
+- **Connectors:**  Some of the Kafka connect available connectors usually have limitations. It's usually rare to find
+  all the required features in a single connector and so is to find documentation for the same.
+- **Fully open-source:**  Firehose is completely open-source while separation of commercial and open-source features is
+  not very structured in Kafka Connect and for monitoring and advanced features, confluent control center requires an
+  enterprise subscription.
+
+#### What problems does Firehose solve?
+
+Every micro-service needs its own sink to be developed for common operations such as streaming data from Kafka to data
+lakes or other endpoints, along with real-time filtering, parsing, and monitoring of the sink.
+
+With Firehose, you don't need to write sink code for every such microservice, or manage resources to sink data from
+Kafka server to your database/service endpoint. Having provided all the configuration parameters of the sink, Firehose
+will create, manage and monitor one for you. It also automatically scales to match the throughput of your data and
+requires no ongoing administration.
+
+#### Can I do any transformations before sending data to sink, for example filtering ?
+
+Yes, Firehose provides JEXL based filters based on the fields in key or message of the Kafka record. Read
+the [Filters](https://github.com/odpf/firehose/blob/main/docs/concepts/filters.md) section for further details.
+
+#### How to optimise parallelism based on input rate of Kafka messages? Does it depend on sink ?
```

Comment:
It might be better to use asysnc consumer. I think we should remove application_thread in later release.

Reply:
@lavkesh Linked to the consumer documentation